### PR TITLE
Footer and Voltsale update 

### PR DESCRIPTION
**Pull request by RudeWolfCoding**

### CODE REVIEW

File: src/components/home/components/volt_sale_card.jsx [hunk 1]
```diff
@@ -23,6 +23,8 @@ const VoltSaleCard = () => {
 
   const [tokenSaleAddress, setTokenSaleAddress] = useState(null)
 
+  const [tokenPriceSelected, setTokenPriceSelected] = useState('')
```
Comment (LiorAgnin) [inline src/components/home/components/volt_sale_card.jsx:26]: @RudeWolfCoding Is it really necessary to separate the to `tokenSaleAddress` & `tokenPriceSelected`?

File: src/components/home/components/volt_sale_card.jsx [hunk 2]
```diff
@@ -174,6 +179,14 @@ const VoltSaleCard = () => {
         <div className='grid-x'>
           <div className='small-24'>
             <NumericalInput label='VOLT' value={tokenAmount} />
+            <div style={{ display: 'flex', width: '100%', justifyContent: 'space-between', marginBottom: '48px', paddingInline: '10px' }}>
```
Comment (LiorAgnin) [inline src/components/home/components/volt_sale_card.jsx:182]: @RudeWolfCoding Please don't use inline styles!

File: src/components/home/components/volt_sale_card.jsx [hunk 3]
```diff
@@ -150,18 +154,19 @@ const VoltSaleCard = () => {
...
-              label='FUSE'
+              label='Fuse'
               value={typedValue}
               onChange={setTypedValue}
               onMax={onMax}
               showMax
             />
           </div>
           <div className='cell small-24 medium-12'>
+            <p style={{ marginBottom: '5px', marginRight: '5px', textAlign: 'right' }}>Balance: {fuseBalance ? Number(fuseBalance).toFixed(4) : 0}</p>
```
Comment (LiorAgnin) [inline src/components/home/components/volt_sale_card.jsx:165]: @RudeWolfCoding Please do not use inline styles!

File: src/components/home/components/volt_sale_card.jsx [hunk 4]
```diff
@@ -120,12 +123,13 @@ const VoltSaleCard = () => {
         style={modalStyle}
         contentLabel='Voltage Info'
       >
+        <h1 style={{textAlign: 'center', fontSize: '25px', paddingBottom: '10px'}}>Some Text</h1>
```
Comment (LiorAgnin) [inline src/components/home/components/volt_sale_card.jsx:126]: @RudeWolfCoding What is `Some Text`?

File: src/components/home/components/volt_sale_card.jsx (hunk 3 above)
Comment (RudeWolfCoding) [reply src/components/home/components/volt_sale_card.jsx]: Changing inline styling into SASS

File: src/components/home/components/volt_sale_card.jsx (hunk 4 above)
Comment (RudeWolfCoding) [reply src/components/home/components/volt_sale_card.jsx]: Placeholder, got updated text from Tomas. Opening a PR soon to update it.

File: src/components/home/components/volt_sale_card.jsx (hunk 1 above)
Comment (RudeWolfCoding) [reply src/components/home/components/volt_sale_card.jsx]: necessary for the volt sale card to show the appropriate calculations of Totals to the user. tokenPriceSelected is getting information from the price select input.

File: src/components/home/components/volt_sale_card.jsx (hunk 2 above)
Comment (RudeWolfCoding) [reply src/components/home/components/volt_sale_card.jsx]: Reworking into SASS

**Pull request merged**